### PR TITLE
chore: change endianess on idCommitment

### PR DESCRIPTION
**Pull request by danisharora099**
ref: waku-org/js-rln#37

### CODE REVIEW

**Comment by danisharora099** [comment]
> LGTM! Did you try registering and interface with nwaku chat2?

Yes, I registered and dialed wakuv2 test node
Screenshot 2022-12-30 at 3 31 24 PM

File: rln-js/index.html [hunk 1]
```diff
@@ -297,6 +297,12 @@ <h4 class="mu1">Messages</h4>
         updateFields();
       };
 
+      const changeEndianess = (hex) => {
+        const bytes = ethers.utils.arrayify(hex);
+        const reversed = bytes.reverse();
+        return ethers.utils.hexlify(reversed);
+      };
+
```
Comment (fryorcraken) [inline rln-js/index.html:305]: This is a lot of manipulation: Uint8Array -> hex string -> bytes (Uint8Array) -> reverse -> hex string.

I would recommend instead:  Uint8Array -> Bigint
and use the bigint in etherjs API moving forward.

You can do that using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView/getBigUint64

**Review by fryorcraken — COMMENTED** [review]
that's fine as a temporary solution but as discussed, the next step would be to have this handled in js-rln library and provide helpful API to the users:
1. API to return id commitment as a bigint
2. API To register membership using etherjs

As tracked with https://github.com/waku-org/js-waku-examples/issues/163

Cc @weboko

**Comment by fryorcraken** [comment]
> > LGTM! Did you try registering and interface with nwaku chat2?
> 
> Yes, I registered and dialed wakuv2 test node

But this was already working before. The difference is that now the id committment is properly registered in the contract so it can be verified with root but we currently verify without roots in JS.

do we verify with roots in nwaku @s1fr0 ?

Would be good to show before/after. the expectatiion is:
- before: verification with roots fails
- after: verification with roots succeed

We can also verify with roots in js-waku actually by using the right API.

File: rln-js/index.html (hunk 1 above)
Comment (danisharora099) [reply rln-js/index.html]: Addressed.

Figured a shorter way:
Uint8Array -> reverse -> pass to ethersjs

**Comment by danisharora099** [comment]
This PR has been superseded by https://github.com/waku-org/js-waku-examples/pull/188 because of a huge rebasing change.

**Comment by fryorcraken** [comment]
> This PR has been superseded by #188 because of a huge rebasing change.

How is a rebase stopping you to deal with it?

rebasing mean that git changes your base commit and re-applies your commits. Your commits are 5 lines changes.

Just:
- Squash your commit in 1 (keep a clean commit tree)
- rebase

**Pull request closed without merging**